### PR TITLE
Added a chat clear command bound to /ch cc

### PR DESCRIPTION
**Pull request by amir9189100**
Still a bit new to sponge, and java in general, but I asked a few people
and they said this idea could be helpful, and I decided to implement it.

### CODE REVIEW

File: src/main/java/com/helion3/darmok/commands/ChannelChatClearCommand.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import org.spongepowered.api.util.command.CommandException;
+import org.spongepowered.api.util.command.CommandResult;
+import org.spongepowered.api.util.command.CommandSource;
+
+import com.google.common.base.Optional;
+import com.helion3.darmok.utils.Format;
+
+public class ChannelChatClearCommand implements CommandCallable {
+    public Optional<CommandResult> process(CommandSource source, String arguments) throws CommandException {
+        for(int i = 0; i < 100; i++){
```
Comment (viveleroi) [inline src/main/java/com/helion3/darmok/commands/ChannelChatClearCommand.java]: Indentation formatting looks a bit off here, please be sure you're checking everything with gradle. Also, can you please put a space after `for` and before the opening curly brace? `for (int i = 0; i < 100; i++) {`.

**Comment by viveleroi** [comment]
Please squash these two commits.

**Comment by viveleroi** [comment]
Why did you close this?

**Comment by amir9189100** [comment]
The idea isn't really that great, people could essentially spam this and avoid any messages from moderators or players trying to get their attention. Letting people clear their own chat could really backfire unless I could implement something to prevent that.

**Pull request closed without merging**